snmp/tests/test_profiles.py

```diff
@@ -1652,6 +1652,16 @@ def test_chatsworth(aggregator):
             'snmp.cpiPduBranchEnergy', metric_type=aggregator.MONOTONIC_COUNT, tags=branch_tags, count=1
         )
 
+    for branch in [1]:
```

Comment:
Do we really need a loop for that?

Reply:
Not really, but it's more consistent with what's done above and will make it easier to add more branches to the tests if needed.